homeassistant/exceptions.py

```diff
@@ -12,6 +12,20 @@
 class HomeAssistantError(Exception):
     """General Home Assistant exception occurred."""
 
+    def __init__(self, *args: object, omit_stack_trace: bool = False) -> None:
+        """Initialize Home Assistant Error.
+
+        When omit_stack_trace is set, a stack trace is logged to
+        the debug logging when available.
```

Comment:
This is misleading; setting `omit_stack_trace` does not automatically influence the logging, it's just a hint to whatever is catching exceptions.